### PR TITLE
Sites Model

### PR DESCRIPTION
**Pull request by athai**
Fixes issue #16.

## Changes in this PR.

- [X] Adds some basic tests for the ``sites`` model
- [X] Implementation of the model 

## Testing this PR.

1. Start up an instance of the StreamWebs docker container
2. Log into the container w/ ``docker-compose run web bash``
3. ``cd`` into /streamwebs_frontend
4. Run ``python manage.py test streawebs/tests/models`` 

### Expected Output.
```
(venv) [root@390b322ed535 streamwebs_frontend]# python manage.py test streamwebs/tests/models/
Creating test database for alias 'default'...
....
----------------------------------------------------------------------
Ran 4 tests in 0.002s

OK
Destroying test database for alias 'default'...

```

### CODE REVIEW

**Comment by Kennric** [comment]
I believe this will correctly test that the model was defined correctly. +1

File: streamwebs_frontend/streamwebs/tests/models/test_site_model.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    def test_fields_exist(self):
+        model = model.get_model('streamwebs', 'site')
+        for field, field_type in self.expected_fields.items():
+            self.assertEqual(
+                field_type, type(model._meta.get_field_by_name(field[0])))
+
+    def test_no_extra_fields(self):
+        fields = Site._meta.get_all_field_names()
+        self.assertEqual(sorted(fields), sorted(self.expected_fields.keys()))
```
Comment (matthewrsj) [inline streamwebs_frontend/streamwebs/tests/models/test_site_model.py]: You shouldn't have to sort python dicts when checking for equality. `assertEqual()` calls `assertDictEqual()` when passed two dicts.

See: https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertDictEqual

**Comment by matthewrsj** [comment]
Consider removing the `sorted()` function calls above. +1 regardless.

**Comment by athai** [comment]
@matthewrsj The tests results in this error if the ``sorted()`` function is removed:
```
FAIL: test_no_extra_fields (test_site_model.SiteTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/streamwebs/streamwebs_frontend/streamwebs/tests/models/test_site_model.py", line 41, in test_no_extra_fields
    self.assertEqual(fields, self.expected_fields.keys())
AssertionError: Lists differ: ['site_name', 'description', '... != ['site_name', 'site_type', 'cr...

First differing element 1:
description
site_type

  ['site_name',
-  'description',
+  'site_type',
   'created',
+  'id',
+  'site_slug',
+  'location',
   'modified',
+  'description']
-  'location',
-  'site_slug',
-  u'id',
-  'site_type']
```

**Comment by matthewrsj** [comment]
@athai I stand corrected. I thought those were two dicts. +1

**Comment by ElijahCaine** [comment]
Looks good to me. +1

**Comment by Kennric** [comment]
Looks good to me, tests pass. +1

**Pull request merged**